twake/frontend/src/app/views/applications/messages/message/parts/MessageAttachments.tsx

```diff
@@ -44,8 +44,7 @@ export default () => {
               files.filter(
                 file =>
                   file.metadata?.source === 'internal' &&
-                  (file.metadata?.thumbnails?.length || 0) > 0 &&
-                  FileUploadAPIClient.mimeToType(file.metadata?.mime || '') === 'image',
```

Comment:
We should still restrict the files that support large preview and the one that doesn't support it. If I push an image and a zip both should be in small view (to avoid misalignment)
So the condition should stay here:
```
file.metadata?.source === 'internal' &&
(file.metadata?.thumbnails?.length || 0) > 0 &&
["image", "video", "slides", "spreadsheet", "document", "pdf"].includes(FileUploadAPIClient.mimeToType(file.metadata?.mime || ''))
```